iroh-resolver/src/content_loader.rs

```diff
@@ -133,6 +144,32 @@ impl FullLoader {
         }
     }
 
+    async fn fetch_dht_record(&self, cid: &Cid) -> Result<iroh_rpc_types::p2p::PeerRecord> {
+        let key_value = "/ipns/"
+            .bytes()
+            .into_iter()
+            .chain(
+                PeerId::from_multihash(*cid.hash())
+                    .unwrap()
+                    .to_bytes()
+                    .into_iter(),
+            )
+            .collect::<Vec<_>>();
+        let key = Key::from(key_value);
+        match self.client.try_p2p() {
+            Ok(p2p) => Ok(p2p
+                .dht_get(&key, Quorum::N(NonZeroUsize::new(16).unwrap()))
```

Comment:
this should be a constant, with some docs on why we use this number

Reply:
Let's even move this parameter to config?

I saw fights in discussion threads about why 16 have been chosen and multiple people had tried to play with the number. Tbh, I didn't find precise explanation how Kubo authors had derived this number and why is it not overkilling. Afaik even in Sybil attack the worst damage will be providing outdated records merely. Intuitively, 16 is too high but I won't bet anything on this claim.

So, I'm not sure how to explain why we use this number

Reply:
yes, config sounds good 👍